vignettes/new-application.Rmd

```diff
@@ -175,7 +176,7 @@ Different parts of the generated application can be customized with a custom yam
 This is a high-functionality set of custom-styled buttons with the built-in 
 ability to download data in different file formats.
 
-*See the downloadFileButton Vignette for more detailed information*
+*See the downloadFile Vignette for more detailed information*
```

Comment:
downloadFileButton is a section in the downloadFile vignette here: https://cran.r-project.org/web/packages/periscope/vignettes/downloadFile-module.html#downloadfilebutton